### PR TITLE
[ISSUE 11714][DOC][spelling mistakes]fix spelling mistakes for doc

### PR DESCRIPTION
**Pull request by hapihu**
There are some spelling mistakes in the document.

```bash
site2/docs/security-jwt.md:260: forwared ==> forwarded
site2/docs/security-kerberos.md:196: preform ==> perform
site2/docs/security-kerberos.md:221: comparision ==> comparison
site2/docs/security-kerberos.md:241: comparision ==> comparison
site2/docs/helm-overview.md:13: compoments ==> components
site2/docs/helm-overview.md:13: seperately ==> separately
site2/docs/helm-overview.md:92: conciously ==> consciously
site2/docs/security-athenz.md:44: encrption ==> encryption
site2/docs/client-libraries-websocket.md:72: Broswer ==> Browser
site2/docs/security-authorization.md:40: creats ==> creates

site2/docs/functions-develop.md:1097: refered ==> referred

site2/docs/developing-binary-protocol.md:415: retreive ==> retrieve
site2/docs/security-tls-transport.md:81: creat ==> create

site2/docs/administration-pulsar-manager.md:129: PostgresSQL ==> postgresql
site2/docs/administration-pulsar-manager.md:144: genrated ==> generated
site2/docs/administration-pulsar-manager.md:145: genrated ==> generated
site2/docs/administration-pulsar-manager.md:150: PostgresSQL ==> postgresql
site2/docs/administration-pulsar-manager.md:165: genrated ==> generated
site2/docs/administration-pulsar-manager.md:170: PostgresSQL ==> postgresql
site2/docs/deploy-bare-metal.md:171: exsiting ==> existing
site2/docs/deploy-bare-metal.md:375: succesfully ==> successfully


site2/docs/client-libraries-java.md:594: follwoing ==> following
site2/docs/client-libraries-java.md:627: follwoing ==> following
site2/docs/client-libraries-java.md:637: follwoing ==> following
site2/docs/security-tls-keystore.md:124: interal ==> internal
site2/docs/security-tls-keystore.md:229: interal ==> internal
site2/docs/concepts-messaging.md:522: dicates ==> dictates
site2/docs/administration-dashboard.md:54: explicitely ==> explicitly
site2/docs/administration-load-balance.md:118: avaliable ==> available
site2/docs/reference-pulsar-admin.md:2426: garanteed ==> guaranteed
site2/docs/reference-pulsar-admin.md:2620: specifed ==> specified
site2/docs/reference-configuration.md:59: readd ==> read
site2/docs/reference-configuration.md:100: ture ==> true
site2/docs/reference-configuration.md:247: implementatation ==> implementation
```

### CODE REVIEW

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ BookKeeper is a replicated log storage system that Pulsar uses for durable stora
 |compactionRate|The rate at which compaction will read entries, in adds per second.|1000|
 |isThrottleByBytes|Throttle compaction by bytes or by entries.|false|
 |compactionRateByEntries|The rate at which compaction will read entries, in adds per second.|1000|
-|compactionRateByBytes|Set the rate at which compaction will readd entries. The unit is bytes added per second.|1000000|
+|compactionRateByBytes|Set the rate at which compaction will read entries. The unit is bytes added per second.|1000000|
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:59]:
```suggestion
|compactionRateByBytes|Set the rate at which compaction reads entries. The unit is bytes added per second.|1000000|
```

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution. Does this affect only master or other versioned docs? 
If latter, could you please help update all affected versions? Thanks

**Comment by hapihu** [comment]
> Thanks for your contribution. Does this affect only master or other versioned docs?
> If latter, could you please help update all affected versions? Thanks

Hi, @Anonymitaet 
Other versions of the document also have these problems.
I ’d like to update all of the affected versions, such as branch-2.8,brabch-2.7, and so on.
Is it ok to handle only all Active branches?
Do I need to create a pull request for each affected version?

**Review by merlimat — APPROVED** [review]

**Comment by sijie** [comment]
@hapihu You can do it in this PR if you prefer.

**Comment by hapihu** [comment]
> @hapihu You can do it in this PR if you prefer.

Ok, I'll finish it as soon as possible

**Comment by Anonymitaet** [comment]
@hapihu please update versioned docs [here](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs) in this PR, thanks

**Comment by hapihu** [comment]
> @hapihu please update versioned docs [here](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs) in this PR, thanks

Ok, sorry, I just found the file location of other version, I will fix it as soon as possible.

**Comment by hapihu** [comment]
@Anonymitaet 
I have update all affected versions.
Would you like to review it for me again.
Thank you very much!

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**